index.html

```diff
@@ -386,6 +386,27 @@
           a representation of the signing date.</dd>
         </dl>
       </section>
+
+      <section>
+        <h2>Revocation Model</h2>
+        <p>A <a>Verifiable Claim</a> may provide revocation information
+        by referring to a revocation scheme by adding the following
+        <a>property</a> to the <a>Verifiable Claims Model</a>:</p>
```

Comment:
(note that all styling was removed from what I typed above.  The 'Verifiable Claims Model' and 'property' should both be hyperlinks)